src/InputWithLabel.js

```diff
@@ -0,0 +1,25 @@
+import React, { useRef, useEffect } from "react";
+
+
+function InputWithLabel({ todoTitle, handleTitleChange, children }) {
+	const inputRef = useRef();
+	useEffect(() => {
+		inputRef.current.focus();
+	});
+	return (
+		<div>
+			<label htmlFor="todoTitle">{children}</label>
+			<input
+				ref={inputRef}
+				value={todoTitle}
+				onChange={handleTitleChange}
+				id="todoTitle"
+				name="title"
+			></input>
+		</div>
```

Comment:
The lesson instructions suggested to use a fragment (`<></>`). Using the `<div>` instead is fine, but there may be times when you don't want to contain everything inside a `<div>`, so make sure you know what a fragment is and when you might use it.